LondonBritishLibrary/orient/BLorient686.xml

```diff
@@ -186,22 +582,24 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                         <list>
 
-                           <item xml:id="a1">
-                              <locus target="#1a30"/>
-                              <quote xml:lang="gez">ገድለ፡ ሰማዕታት፡ ዘቅ' ፡ መ' ፡ ዓለም። </quote>
-                              <quote xml:lang="en">the Acts of Martyrs, belonging to the Holy Saviour of the World</quote>
-
-                              <p>i. e., to the <placeName ref="INS00100MadhaneAlam">church</placeName>
-                           </p>
-
-
-                              <desc type="OwnershipNote">The note stating that the manuscript belongs, to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
-                           </desc>
-
+                          <item xml:id="a1">
+                          <locus target="#1r"/>
+                          <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                          <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                          <q xml:lang="gez">ገድለ፡ ሰማዕታት፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+                          </item>
 
-                           </item>
 
+<item xml:id="a2">
```

Comment:
I would encode both these items with an "e" ID, because they are not later additions.

Reply:
I will keep that in mind!